start.sh

```diff
@@ -16,8 +16,12 @@ if [ $CURRENT_VERSION != "v$DESIRED_VERSION" ]; then
       echo "Installing node $DESIRED_VERSION..."
       nvm install $DESIRED_VERSION
     fi
+  elif [ -f /usr/local/bin/n ]; then
```

Comment:
Not sure how these changes ended up in here, perhaps when rebasing on latest master? Just so you know, when we add the `ready to merge` label, Mergify will automatically apply your commits onto latest master, so you don't need to manually keep merging in latest master once you've submitted the PR.

Reply:
sorry that was my bad, i use n and it would start in random node versions depending on what app i was working with last i'll revert it